datarepo-clienttests/src/main/resources/configs/basicexamples/SnapshotScaleDelete.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
This file was added but I don't see code that references it. Does the code that calls your test use the test class name to find the json file?

Reply:
Yes, the test runner code handles that